contracts/LockDealNFT/LockDealNFT.sol

```diff
@@ -40,13 +56,12 @@ contract LockDealNFT is LockDealNFTModifiers, ILockDealNFTEvents {
         notZeroAddress(provider)
         returns (uint256 poolId)
     {
+        require(amount > 0, "amount must be greater than 0");
```

Comment:
We have the `notZeroAmount` modifier. Let's add it to the list of modifier calls.